<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureWidget</class>
 <widget class="QWidget" name="TemperatureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>1028</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; text-decoration: underline;&quot;&gt;Temperature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="iNum">
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="iPrefix">
       <property name="currentIndex">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>yotta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zetta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>exa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>peta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>giga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mega</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kilo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hecto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>deka</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>deci</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>centi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>milli</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>micro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nano</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>femto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>atto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zepto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yocto</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="iUnit">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Kelvin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Celcius</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delisle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Newton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rankine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Réaumur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rømer</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="downIcon">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources.qrc">:/icons/down.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="oPrefix">
       <property name="currentIndex">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>yotta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zetta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>exa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>peta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>giga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mega</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kilo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hecto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>deka</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>deci</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>centi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>milli</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>micro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nano</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>femto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>atto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zepto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yocto</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="oUnit">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Kelvin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Celcius</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delisle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fahrenheit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Newton</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rankine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Réaumur</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rømer</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="oNum">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="keyboardSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>216</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
